gravitee-apim-console-webui/src/organization/configuration/console/org-settings-general.component.ts

```diff
@@ -189,10 +189,92 @@ export class OrgSettingsGeneralComponent implements OnInit, OnDestroy {
 
     const formSettingsValue = this.formSettings.getRawValue();
 
-    const settingsToSave = merge(cloneDeep(this.settings), formSettingsValue);
+    const updatedSettingsPayload = {
```

Comment:
I understand what you did but would it work with something like that ?

```suggestion
    const updatedSettingsPayload = merge({}, this.settings, formSettingsValue);
```

Reply:
It gives us control in future if we want to modify something.